Detectors/GlobalTracking/src/MatchITSTPCQC.cxx

```diff
@@ -259,7 +265,11 @@ void MatchITSTPCQC::run(o2::framework::ProcessingContext& ctx)
         }
         if (mMapTPCLabels.find(lbl) == mMapTPCLabels.end()) {
           o2::MCTrack const* mcParticle = mcReader.getTrack(lbl);
-          if (isPhysicalPrimary(mcParticle)) {
+          int source = lbl.getSourceID();
+          int event = lbl.getEventID();
+          const std::vector<o2::MCTrack>& pcontainer = mcReader.getTracks(source, event);
+          const o2::MCTrack& p = pcontainer[itrk];
+          if (MCTrackNavigator::isPhysicalPrimary(p, pcontainer)) {
```

Comment:
it looks ok. Just that probably one should choose between either `mcParticle` or `p`.
One could just say `MCTrackNavigator::isPhysicalPrimary(*mcParticle, pcontainer)` and do not add the line
`const o2::MCTrack& p = pcontainer[itrk];` (or vice-versa)

Reply:
@sawenzel thank you for your review. I had just deleted the  mcParticles object since it remained unused (it was only called in the previous definition of isPhysicalPrimary).